src/app/components/ItemPage/ItemPage.jsx

```diff
@@ -13,16 +13,16 @@ class ItemPageRegular extends React.Component {
     // const ebscoLink = record.PLink;
     // const bibRecord = record.RecordInfo.BibRecord;
     const title = record.title[0];
-    const authors = record.contributor && record.contributor.length ? 
-      record.contributor.map((author, i) => (<a href="#" key={i}>{author}; </a>))
+    const authors = record.contributor && record.contributor.length ?
+      record.contributor.map((author, i) => (<Link to={{ pathname: '/search', query: { q: `contributor:"${author}"` } }} key={i}>{author}</Link>))
       : null;
-    const authorsString = record.contributor && record.contributor.length ? 
-      record.contributor.map((author, i) => (`<a href="#" key={i}>${author}; </a>`))
+    const authorsString = record.contributor && record.contributor.length ?
+      record.contributor.map((author, i) => (`<a href="/search?q=${encodeURIComponent('contributor:"'+author+'"')}" key=${i}>${author}</a>`))
```

Comment:
I've been coding this in ES6 syntax. The ES6 string interpolation can be used here instead of the `+`. It's not a big deal since it's just syntax, but the consistency could help down the road.